recipes/maxentscan/build.sh

```diff
@@ -0,0 +1,15 @@
+target=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $target
+mkdir -p $PREFIX/bin
+
+#change hardcoded directory of splicemodels
+sed -i.bak "s@\"splicemodels/\"@\"$target/splicemodels/\"@" *.pl
+
+cp score3.pl $target/maxentscan_score3.pl
+cp score5.pl $target/maxentscan_score5.pl
```

Comment:
If you change the name, which is reasonable, you should add that to extra->notes (see e.g. [here](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/gatk/meta.yaml)).